src/app/iteration/iteration.component.html

```diff
@@ -179,12 +179,6 @@ <h3 class="flex-1">Iterations</h3>
     </div>
   </div>
 </div>
-<div *ngIf="loggedIn"
-  class="create-iteration-bar pointer"
-  (click)="modal.openCreateUpdateModal('create');">
-  <span><i class="pficon-add-circle-o fa-icon"></i>
```

Comment:
We would need this class (**pficon-add-circle-o fa-icon**) since a functional test is referencing on that. @naina-verma please have a look.